sphinx/templates/quickstart/conf.py_t

```diff
@@ -141,7 +141,13 @@ latex_documents = [
     (master_doc, '{{ project_fn }}.tex', u'{{ project_doc_texescaped_str }}',
      u'{{ author_texescaped_str }}', 'manual'),
 ]
-
+{% if language|default('').split('_')[0] == 'zh' %}
+# Render Chinese LaTex defaults to xelatex+ctex as it works in most cases
+# Leave it configurable here for user customization
+if language.split('_')[0] == 'zh':
+    latex_engine = 'xelatex'
+    latex_elements = {'usepackages':'\\usepackage{ctex}','polyglossia':''}
```

Comment:
`usepackage` is an internal variable, so we can't say this always works fine.
It might be overwritten by any extensions. (It is used for `Sphinx.add_latex_package() API.)

How about this? I believe it might work fine.
```
latex_elements = {'polyglossia': '\\usepackage{ctex}'}
```